src/demangle.h

```diff
@@ -77,7 +77,7 @@ _START_GOOGLE_NAMESPACE_
 // Demangle "mangled".  On success, return true and write the
 // demangled symbol name to "out".  Otherwise, return false.
 // "out" is modified even if demangling is unsuccessful.
-bool GOOGLE_GLOG_DLL_DECL Demangle(const char *mangled, char *out, int out_size);
```

Comment:
So I am curious:

* What is the definition of `GOOGLE_GLOG_DLL_DECL` on your system?  I would have naively assumed that it's for Windows only (because: DLL) and that it's empty elsewhere.
* What is the error message that you get?